fiftyone/utils/sam2.py

```
@@ -207,133 +183,81 @@ class SegmentAnything2ImageModel(fosam.SegmentAnythingModel):
-        model = model.to(self._device)
-        if self.using_half_precision:
-            model = model.half()
+    def _load_predictor(self):
+        return _SAM2Predictor(self._model)
 
-        model.eval()
+    def _load_auto_generator(self):
+        kwargs = self.config.auto_kwargs or {}
+        return samg.SAM2AutomaticMaskGenerator(self._model, **kwargs)
 
+    def _load_model(self, config):
```

Ruff (0.15.9)

[warning] Missing return type annotation for private function `_load_model`

(ANN202)